### DISCUSSION TITLE
Is the licence file correct?

### DISCUSSION THREAD

**Discussion by MeGA-ct** [containers/podman#18646] [Q&A] [answered]
In line 189, I see there is no year nor name of the copyright owner.
https://github.com/containers/podman/blob/44807eabf1790076c071270fc9b3d0958f280bb9/LICENSE#L189
Is that correct?

Kind regards.

**Comment by Luap99** [comment]
Looks like it was just copied from the template but we never added the real info there.
@TomSweeneyRedHat @rhatdan PTAL

**Reply by afbjorklund** [reply]
That is just the boilerplate copyright, of the generic license text

> To apply the Apache License to your work, attach the following boilerplate notice, with the fields enclosed by brackets "{}" replaced with your own identifying information. (Don't include the brackets!) The text should be enclosed in the appropriate comment syntax for the file format.

**Comment by richardfontana** [comment, marked as answer]
The current approach by Podman is more correct, in a sense, since it matches what was standard practice in Apache License 2.0-using projects up until several years ago and I am pretty sure it follows how the license appendix was originally intended to be used. The Apache-2.0 appendix is probably modeled on the "How to Apply" appendix in the much older GPLv2, which itself contains what was meant to be a joke copyright notice. 

In the case of the Apache License 2.0, the idea that you're supposed to instantiate the template in the appendix originated in some project or group of projects at that time, as (I would argue) an erroneous understanding of how the license was supposed to be used, and somehow caught on among a number of other projects, such that it's difficult to argue that the newer approach is incorrect. 

Personally, being a bit of a traditionalist, I'd prefer it if the suggestion made by @MeGA-ct was not adopted, but it's hard to argue that it matters much one way or the other.

**Reply by MeGA-ct** [reply]
Thx for answering.

I think I understand why is kept that way, and I don't want to trigger any unnecessary change.
